.eslintignore

```diff
@@ -0,0 +1 @@
+.md
```

Comment:
I think it's the atom package thats just out of date. Haven't seen it try to lint it elsewhere.